### PR TITLE
[platform_tests] Add test to check memory and cpu usage

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Add test `platform_tests/test_cpu_memory_usage.py`:
1. Check DUT memory usage don't exceeds threshold.
For the following platforms, threshold is 75%.
x86_64-cel_e1031-r0
x86_64-arista_7050_qx32s
x86_64-arista_7050_qx32
For others, threshold is 60%.

2. Check process cpu percent don't persistently exceeds threshold: 50%.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### CODE REVIEW

File: ansible/library/monit_process.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            interval=interval, iterations=(iterations + 1)
+            )
+        rc, stdout, _ = module.run_command(args=cmd)
+
+        monit_results = []
+        proc_section = False
+        mem_re = re.compile(
+            (r"^KiB Mem\s+:\s+(?P<total>\d+)\s+total,\s+(?P<free>\d+)"
+             r"\s+free,\s+(?P<used>\d+)\s+used,\s+\d+\s+buff/cache$")
+        )
```
Comment (bingwang-ms) [inline ansible/library/monit_process.py:52]: The output of top is different in different platforms. On dx010-4, I got following output
```
root@str-dx010-acs-4:~# top -d 12 -n 1 -b
top - 09:28:21 up 18:32,  2 users,  load average: 2.19, 2.41, 2.28
Tasks: 264 total,   1 running, 233 sleeping,   0 stopped,  30 zombie
%Cpu(s): 14.7 us, 30.7 sy,  0.0 ni, 54.7 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
MiB Mem :   3923.4 total,    504.7 free,   2143.7 used,   1275.1 buff/cache
MiB Swap:      0.0 total,      0.0 free,      0.0 used.   1481.0 avail Mem 
```
The Mem is in **MiB** rather that **KiB**. Can we use other method to retrieve mem usage (like free)?

Comment (lolyu) [reply ansible/library/monit_process.py:52]: Updated, using `-E` to enforce uniform output size unit.

**Review by bingwang-ms — APPROVED** [review]

**Comment by lolyu** [comment]
Retest this please

**Comment by lolyu** [comment]
Retest this please

**Review by andrewchoi5 — APPROVED** [review]

**Comment by lolyu** [comment]
Retest this please

**Comment by lolyu** [comment]
Retest this please

**Pull request merged**